docs/topics/Voice_Connections.md

```diff
@@ -97,17 +208,55 @@ Voice data sent to discord should be encoded with [Opus](https://www.opus-codec.
 
 ### Speaking
 
-To notify clients that you are speaking or have stopped speaking, send an OP5 Speaking payload.
+To notify clients that you are speaking or have stopped speaking, send an [OP 5 Speaking](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) payload:
 
 ###### Example Speaking Payload
 
 ```json
 {
 	"speaking": true,
-	"delay": 0
+	"delay": 0,
+	"ssrc": 1
 }
 ```
 
+>warn
+>You must send at least one [OP 5 Speaking](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) payload before sending voice data, or you will be disconnected with an invalid SSRC error.
+
 ### Voice Data Interpolation
 
-When there's a break in the sent data, the packet transmission shouldn't simply cut off but rather send five frames of silence (`0xF8, 0xFF, 0xFE`) before stopping to avoid unintended Opus interpolation with following transmissions.
+When there's a break in the sent data, the packet transmission shouldn't simply stop. Instead, send five frames of silence (`0xF8, 0xFF, 0xFE`) before stopping to avoid unintended Opus interpolation with subsequent transmissions.
+
+## Resuming Voice Connection
+
+When your client detects that its connection has been severed, it should completely close the connection and open a new one following the same flow from [Connecting](#DOCS_VOICE_CONNECTIONS/connecting-to-voice). Once the new connection has been opened, your client should send an [OP 7 Resume](#DOCS_VOICE_CONNECTIONS/voice-events-voice-op-codes) payload:
```

Comment:
I don't think this is right. The `connecting-to-voice` section talks about getting server information and identifying, etc. When you resume, you already have all that information and just need to send a resume after you reopened the websocket with the voice server.